src/Command/Command.php

```diff
@@ -2,15 +2,15 @@
 
 namespace Drupal\Console\Command;
 
-use Symfony\Component\Console\Command\Command as BaseCommand;
+use Consolidation\AnnotatedCommand\AnnotatedCommand;
 use Symfony\Component\Console\Helper\HelperSet;
 use Drupal\Console\Helper\HelperTrait;
 
 /**
  * Class Command
  * @package Drupal\Console\Command
  */
-abstract class Command extends BaseCommand
+abstract class Command extends AnnotatedCommand
```

Comment:
Drupal\Console\Command now extends from AnnotatedCommand.  AnnotatedCommand extends Symfony\Component\Console\Command\Command, and will behave exactly like a Symfony/Console Command unless there are annotations present that configure the command.  Even when annotations are used, the base class will still operate the same way, calling `execute` with `$input` and `$output` variables when the command is used.